### PR TITLE
SpreadElement

### PR DESCRIPTION
**Pull request by hzoo**
~~currently broken - just pushing wip~~

ESTree: https://github.com/estree/estree/blob/master/es6.md#expressions

### CODE REVIEW

File: test/lib/elements/types/SpreadElement.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        expect(elements[0].argument.name).to.equal('a');
+
+        var args = parseAndGetExpression('f(... a)').arguments;
+        expect(args[0].type).to.equal('SpreadElement');
+        expect(args[0].argument.type).to.equal('Identifier');
+        expect(args[0].argument.name).to.equal('a');
+    });
+
+    it('should accept a single identifier with a comment in between', () => {
+        var elements = parseAndGetExpression('[... /* adsf */ a]').elements;
```
Comment (hzoo) [inline test/lib/elements/types/SpreadElement.js]: Getting

```
 1) SpreadElement should accept a single identifier with a comment in between:
     Error: Expression expected but "CommentBlockComment" found
      at ElementAssert._passExpressionInBraces (src/elements/ElementAssert.js:268:19)
      at ElementAssert.passExpression (src/elements/ElementAssert.js:223:21)
```

Comment (mdevils) [inline test/lib/elements/types/SpreadElement.js]: @hzoo please update to new `master`. I fixed `babel`-update issues there.

**Comment by hzoo** [comment]
Rebased - I think I should add tests for ArrayExpression/CallExpression?

**Comment by mdevils** [comment]
Yes, using spread with other arguments tests would be useful.

**Comment by mdevils** [comment]
Great job!

**Comment by hzoo** [comment]
Ok will make another PR for those

**Comment by hzoo** [comment]
Also this isn't as bad as I thought haha :smile:

**Pull request merged**